.github/workflows/test.yml

```diff
@@ -10,10 +10,12 @@ on:
   push:
     branches:
       - develop
+      - develop-v1
```

Comment:
Note that after the release when we merge `trunk` back into "regular" `develop` that we'll likely want to remove this `develop-v1` reference.

Reply:
> Note that after the release when we merge `trunk` back into "regular" `develop` that we'll likely want to remove this `develop-v1` reference.

Yep, I'll create a ticket for the 2.0.0 release to remove these prior to the release. I'll keep them around until then to allow for a 1.8.x or above if needs be.